entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+poetry run gunicorn -b "0.0.0.0:${PORT}" "todo_app.app:create_app()"
```

Comment:
Your prod image now **requires** a PORT env var. You could make it optional (so that your README's docker run command is still accurate) by using `${PORT:-5000}` or by putting `ENV PORT=5000` in your Dockerfile.